packages/utils/src/utils/normalize.ts

```diff
@@ -6,7 +6,7 @@ import upperFirst from 'lodash.upperfirst';
  *
  * Example: ds-token.test becomes DsTokenTest
  */
-export function normalizeName(rawName: string): string {
+export const normalize = (fileName: string): string => {
```

Comment:
How come you opted to shorten the function name? I think for something as broad as normalisation, a more verbose and specific nomenclature is more appropriate here. Additionally, the logic it is referencing has quite a specific use case.

Reply:
Good question. What's your suggestion?

Reply:
@Dhaiwat10 Good one. I was hesitant about writing the full `String` suffix, but I'm cured now. 😅 

@danielbate Let me know if you have a better suggestion.